scripts/superdesk-authoring/authoring.js

```diff
@@ -102,7 +102,7 @@
     }
 
     function stripHtml(item) {
-        var fields = ['headline', 'abstract'];
```

Comment:
@naomiaro  I think this change can go ahead, we'll handle the non-ninjs formatters.